README.md

```diff
@@ -1 +1,121 @@
-# pkid
\ No newline at end of file
+# PKID
+
+PKID is a public Key Indexed Datastore. You can save plain or encrypted data in a public key index; as long as you are the owner of the secret corresponding to that public key.
+
+## Routes
+
+### Set document
+
+```api
+POST /set/{pk}/{project}/{key}
```

Comment:
maybe we can do it bit more RESTish, so `POST {pk}/{project}/{key}` instead of `/set` prefix